lib/pluginmanager/install.rb

```diff
@@ -44,9 +44,11 @@ def validate_cli_options!
   # Check if the specified gems contains
   # the logstash `metadata`
   def verify_remote!(gems)
+    options = { :rubygems_source => gemfile.gemset.sources }
```

Comment:
messing with the sources should be hidden inside the bundler class, the install class doesn't need that kind of information.